plugins/gaming.py

```diff
@@ -119,21 +119,89 @@ def dice(text, notice):
     return "{} ({})".format(total, ", ".join(rolls))
 
 
+choice_split_re = re.compile(r'\s*(?:, ?or|,|or)\s*', re.I)  # regex for splitting choices for choose command
+
+
 @hook.command
 def choose(text, event):
     """<choice1>, [choice2], [choice3], etc. - randomly picks one of the given choices
     :type text: str
     """
-    choices = re.findall(r'([^,]+)', text.strip())
+    choices = choice_split_re.split(text.strip())
+
     if len(choices) == 1:
-        choices = choices[0].split(' or ')
-        if len(choices) == 1:
-            event.notice_doc()
-            return
+        event.notice_doc()
+        return
 
     return random.choice([choice.strip() for choice in choices])
 
 
+# array of digits/characters indexed by int value
+val_char_map = [chr(x) for x in
```

Comment:
This is way overcomplex for what it does. try `list(string.digits + string.ascii_lowercase)` instead. You will need to import string

```
>>> [chr(x) for x in [c for r in (range(ord('0'), ord('9') + 1), range(ord('a'), ord('z') + 1)) for c in r]] == list(string.digits + string.ascii_lowercase)
True
```